lib/lita/handlers/animate.rb

```diff
@@ -12,27 +14,33 @@ def self.default_config(handler_config)
       end
 
       def fetch(response)
```

Comment:
how do you feel about trying to break apart this method? It's a bit of a beast!

you could probably parcel off some methods into single-responsibility methods like an image randomizer, and maybe a response handler?